walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/resolvers/OpenIdURLRequestResolver.kt

```diff
@@ -39,7 +41,11 @@ internal class OpenIdURLRequestResolver(val libraryConfiguration: LibraryConfigu
         if (verifiedIdRequestInput !is VerifiedIdRequestURL) throw UnSupportedVerifiedIdRequestInputException(
             "Provided VerifiedIdRequestInput is not supported."
         )
-        return resolveOpenId4VCIRequest(verifiedIdRequestInput)
+        if (libraryConfiguration.isPreviewFeatureEnabled(PreviewFeatureFlags.FEATURE_FLAG_OPENID4VCI_ACCESS_TOKEN)
+            || libraryConfiguration.isPreviewFeatureEnabled(PreviewFeatureFlags.FEATURE_FLAG_OPENID4VCI_PRE_AUTH)
+        )
+            return resolveOpenId4VCIRequest(verifiedIdRequestInput)
+        return OpenIdResolver.getRequest(verifiedIdRequestInput.url.toString(), preferHeader)
```

Comment:
Comment left on the other PR but I had to change this in oct24rc for extensions to parse requests.

Reply:
We need this check to determine what protocol to use while fetching the request. But moved it to a different location (right before fetching the request).